src/login/logind-core.c

```diff
@@ -55,6 +55,7 @@ void manager_reset_config(Manager *m) {
         m->idle_action = HANDLE_IGNORE;
 
         m->runtime_dir_size = physical_memory_scale(10U, 100U); /* 10% */
+        m->runtime_dir_inodes = m->runtime_dir_size / 4096; /* 4k per inode */
```

Comment:
I think this would be safer with DIV_ROUND_UP(), no? so that runtime_dir_size=3000 results in one inode at least...